modules/nw-aws-load-balancer-with-outposts.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+// * networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-aws-load-balancer-with-outposts_{context}"]
+= AWS Load Balancer Operator and AWS Outposts
+
+AWS Load Balancer Operator can work with AWS Outposts. Ingress resources must be annotated with the either regular subnets or outposts subnets (but not both).
```

Comment:
```suggestion
AWS Load Balancer Operator can work with AWS Outposts. Ingress resources must be annotated with the either outposts or regular subnets (but not both).
```
Since we are talking about the AWS Outposts support.

Reply:
TY